packages/web/src/scss/components/Modal/README.md

```diff
@@ -1,6 +1,6 @@
 # Modal
 
-## Composed Modal (DEPRECATED)
+## Composed Modal
```

Comment:
@adamkudrna I think that this heading is not necessary here. Also, there is another `Modal` subheading a few lines later. Do we need to alter this documentation more or it is just fine?

Reply:
The Modal subheading is for the Modal subcomponent. I agree we can remove this heading and then we can increase the level of the rest of the headings.

Reply:
done